lib/client.go

```diff
@@ -311,6 +311,24 @@ func GetUpdateVariablePayload(organization, workspaceName, variableID string, tf
 `, variableID, tfVar.Key, tfVar.Value, tfVar.Hcl, tfVar.Sensitive, organization, workspaceName)
 }
 
+// OrganizationExists returns whether an organization with the given name exists
+func OrganizationExists(organization string) (bool, err) {
+	if organization == "" {
+		return false, fmt.Errorf("OrganizationExists: organization is required")
+	}
+	u := NewTfcUrl(fmt.Sprintf(
+		"/organizations/%s",
+		organization,
+	))
```

Comment:
Small detail: it would look cleaner, in my opinion, to do this:
```suggestion
	u := NewTfcUrl("/organizations/" + organization)
```